lib/rex/post/meterpreter/ui/console/command_dispatcher/android.rb

```diff
@@ -527,6 +527,7 @@ def cmd_send_sms(*args)
         dest = val
       when '-t'
         body = val
+        body.gsub!('\n',"\n")
```

Comment:
This might need a comment explaining Ruby's quoting and escaping rules.

Reply:
Good point, thanks!